CHANGELOG.rst

```diff
@@ -1,9 +1,17 @@
 =================================
-vmware.vmware_rest Release Notes
+vmware.vmware\_rest Release Notes
 =================================
 
 .. contents:: Topics
 
+v4.0.1
+======
+
+Minor Changes
+-------------
```

Comment:
```suggestion
Bugfixes
--------
```

It's a little bit weird if `changelog/changelog.yaml` says it's a bugfix and `CHANGELOG.rst` says it's a minor change ;-)

Apart from this LGTM!